pom.xml

```diff
@@ -146,24 +188,6 @@
         </dependency>
     </dependencies>
 
-    <profiles>
```

Comment:
Jeg ser du fjerner profilene i maven for unit-test og integration-test. Trengs de ikke i Junit5 ?

Reply:
Jeg brukte profilene til surefire og failsafe, men med @Tag fra Junit5 så ser det ikke ut til å være nødvendig. Jeg kunne bruke strengen fra annotasjonen direkte i <groups>